handlers/graph_test.go

```diff
@@ -0,0 +1,346 @@
+package handlers
+
+import (
+	"context"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/gorilla/mux"
+
+	"github.com/stretchr/testify/assert"
+
```

Comment:
imports should have third party deps in the middle and our sws imports at the bottom with blank line between section. totally trivial and only triggering my ocd

Reply:
i will fix this on a following PR (the tests need a change based on another pending change)